homeassistant/components/flume/const.py

```diff
@@ -47,3 +48,9 @@
 BRIDGE_NOTIFICATION_KEY = "connected"
 BRIDGE_NOTIFICATION_RULE = "Bridge Disconnection"
 NOTIFICATION_LEAK_DETECTED = "Flume Smart Leak Alert"
+
+
+BINARY_SENSOR_DESCRIPTION_CONNECTED = BinarySensorEntityDescription(
```

Comment:
Please keep the entity descriptions in each entity platform. They are platform details.